text/????-separate-semantic-conventions.md

```diff
@@ -0,0 +1,115 @@
+# Separate Semantic Conventions
+
+Move Semantic Conventions outside of the main Specificaiton and version them
+separately.
+
+## Motivation
+
+We need to allow semantic conventions to evolve independently of the overall
+OpenTelemetry specification. Today, any breaking change in a semantic convention
+would require bumping the version number of the entirety of the OpenTelemetry
+specification.
+
+## Explanation
+
+A new github repository called `semantic-conventions` would be created in the
+OpenTelemetry organization.
+
+This would have the following structure:
+
+- Boilerplate files, e.g. `README.md`, `LICENSE`, `CODEOWNERS`, `CONTRIBUTING.md`
+- `Makefile` that allows automatic generation of documentation from model.
+- `model/` The set of YAML files that exist in
+  `{specification}/semantic_conventions` today.
+- `docs/` A new directory that contains human readable documentation for how to
+  create instrumentation compliant with semantic conventions.
+  - `resource/` - Contents of `{specification}/resource/semantic_conventions`
+  - `trace/` - Contents of `{specification}/trace/semantic_conventions`
+  - `metrics/` - Contents of `{specification}/metrics/semantic_conventions`
+  - `logs/`- Contents of `{specification}/logs/semantic_conventions`
+  - `schemas/` - A new location for [Telemetry Schemas](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/schemas/README.md)
+    to live. This directory will be hosted at 
+    `https://opentelemetry.io/semantic_conventions/schemas/`
+
+Existing semantic conventions in the specification would be marked as
+deprecated, with documentation denoting the move.
+
+There will also be the following exceptions in the specification:
+
+- Semantic convetions used to implement API/SDK details will be fully specified
+  and will not be allowed to change in the Semantic Convention directory.
+  - Error/Exception handling will remain in the specification
+  - SDK configuration interaction w/ semantic convention will remain in the
+    specifcation. Specifically `service.name`.
+- The specification may elevate some semantic conventions as necessary for
+  compatibility requirements, e.g. `service.instance.id` and 
+  [Prometheus Compatibility](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md).
+
+## Internal details
+
+The following process would be used to ensure semantic conventions are
+seamlessly moved to their new location.
+
+- A new repository `opentelemetry/semantic_conventions` will be constructed with
```

Comment:
I suggest making a proposal for the approvals structure as part of this PR, since you can't setup a repo without approvers, and I see cleaning up of approvers as one of the main benefits of this OTEP.